testsuite/features/core/proxy_branch_network.feature

```diff
@@ -21,6 +21,13 @@ Feature: Setup SUSE Manager for Retail branch network
     And I manually install the "bind" formula on the server
     And I synchronize all Salt dynamic modules on "proxy"
 
+@proxy
+@private_net
+  # WORKAROUND
+  Scenario: Remove dhcp packages on the proxy
+    When I remove package "dhcp dhcp-client" from this "proxy"
+  # End of WORKAROUND
```

Comment:
Be sure the Build Validation code is also working correctly if the same case comes up there.

Reply:
It shouldn't be necessary. This fix is intended as quick workaround for temporary issue. At the end (and I hope quickly) there are expected to be same package versions at JeOS level and also in the repos.